examples/plot_rgb.py

```diff
@@ -0,0 +1,168 @@
+"""
+Plot Combinations of Raster Bands Using EarthPy
```

Comment:
This is a good start. i think we should add a section on using the extent parameter. please look at the parameters and make sure the key ones are covered in the vignette!

Reply:
Added this using the crop vignette as a starting point.